index.html

```diff
@@ -14,6 +15,184 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  <!--"container-all" is the same on each HTML file-->
+  <div class="container-all">
+    <header>
+      <section class="header-all">
+        <div class="header-top">
+          <h1>James Basile's Resume</h1>
+        </div>
+        <div class="header-bottom">
+          <nav class="header-nav">
+            <a href="index.html">Professional Summary</a>
+            <p>-</p>
+            <a href="#">Education</a>
+            <p>-</p>
+            <a href="#">Work History</a>
+            <p>-</p>
+            <a href="#">Skills/Additional Information</a>
+            <p>-</p>
+            <a href="#">Print Resume</a>
+          </nav>
+        </div>
+      </section>
+    </header>
+    <!-- header image below nav-->
+    <div class="summary-top-img">
+      <p>THIS SHOULD BE AN IMAGE</p>
```

Comment:
I"m glad you're using a reminder like this. Too many people think "I'll totally remember to come back to this" and don't. So good job on making sure you won't forget to go back and update this bit of code by putting this in here